cmd/minikube/cmd/start_flags.go

```diff
@@ -146,6 +146,7 @@ const (
 	staticIP                = "static-ip"
 	gpus                    = "gpus"
 	autoPauseInterval       = "auto-pause-interval"
+	preloadSrc              = "preload-src"
```

Comment:
--preload-src is not a good name for users. We want to use complete words to make the flag more clear. For common flags we provide short option to make it easy to type, but flags that will almost never be used should user clear names.

Change to --preload-source or --preload-provider?

Reply:
thats a good idea !